pkg/cloud/volume_limits.go

```diff
@@ -138,11 +138,17 @@ func GetDedicatedLimitForInstanceType(it string) int {
 	}
 }
 
-func GetNVMeInstanceStoreVolumesForInstanceType(it string) int {
-	if v, ok := nvmeInstanceStoreVolumes[it]; ok {
-		return v
+func GetReservedSlotsForInstanceType(it string) int {
```

Comment:
Would you add a [Go Doc Comment](https://tip.golang.org/doc/comment) above this function that describes what it's doing? 

Something like:
```
// GetReservedSlotsForInstanceType calculates how many attachment slots are already used up by other devices. These devices include Instance Store Volumes and GPUs
```

It's good practice to provide a go doc comment for any exported (capitalized) name so that those who import the package can easily see what it is used for.

For example, IDEs like VsCode and GoLand are able to parse out these go docs to provide on-hover descriptions of the exported function/const.